packages/lodestar/src/network/reqresp/score.ts

```diff
@@ -12,6 +12,14 @@ const libp2pErrorCodes = {
   ERR_UNSUPPORTED_PROTOCOL: "ERR_UNSUPPORTED_PROTOCOL",
 };
 
+/**
+ * Multi stream select error code
+ * https://github.com/multiformats/js-multistream-select/blame/cf4e297b362a43bde2ea117085ceba78cbce1c12/src/select.js#L50
+ */
+const multiStreamSelectErrorCodes = {
+  protocolSelectionFailed: "protocol selection failed",
```

Comment:
once this error happens, it'll happen all the time and the peer is useless to us. It's similar to the `ERR_UNSUPPORTED_PROTOCOL` error below